Source/WebKit/UIProcess/haiku/WebView.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2013 Intel Corporation. All rights reserved.
```

Comment:
Previous copyright should be kept if the file was initially a copy of the work done for another platform.